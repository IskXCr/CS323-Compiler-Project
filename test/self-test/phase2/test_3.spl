struct AA {
    struct BB b;
};

struct BB {};

int func()
{
    struct AA a;
    a.b.c;
    return;
}