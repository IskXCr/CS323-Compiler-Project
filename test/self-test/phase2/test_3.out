Error type 8 at line 11: incompatible return type
Error type 13 at line 13: accessing an undefined structure member
